src/FffGcodeWriter.cpp

```diff
@@ -2524,22 +2523,22 @@ bool FffGcodeWriter::processSupportInfill(const SliceDataStorage& storage, Layer
                                    support_line_width, support_line_distance_here, current_support_infill_overlap - (density_idx == max_density_idx ? 0 : wall_line_count * support_line_width),
                                    infill_multiplier, support_infill_angle, gcode_layer.z, support_shift,
                                    max_resolution, max_deviation,
-                                   (density_idx == max_density_idx ? wall_line_count : 0), infill_origin, support_connect_zigzags,
+                                   0 /*walls are generated somewhere else, so their layers aren't vertically combined*/, infill_origin, support_connect_zigzags,
```

Comment:
`constexpr` the wall line count of 0

Reply:
How is that different?

Reply:
Readability there are a lot of arguments in the function generate. Most IDE display the argument names but not all.
Can't hurt

Reply:
Haha, already decided to fix it as 'readability' after all before reading this comment :-)